pkg/tempo/instance.go

```diff
@@ -81,60 +79,30 @@ func (i *Instance) Stop() {
 }
 
 func (i *Instance) stop() {
```

Comment:
Having a function `Stop` and `stop` seems confusing. Could you clarify the difference? We can't rename `Stop` (part of the OTel interface) but maybe `stop` can be described as `stopApp` or `shutdownApp`.

Basically: why would you want to call `stop` instead of `Stop`?

Reply:
Yea, you're right. Ideally you only want to call the exported function, the other one is for internal use only. But it's confusing, changed in 383c8c1857e93f39ef7d379d063cbdb4bbdf2995.